site/content/en/docs/Guides/Client SDKs/rust.md

```diff
@@ -46,6 +46,15 @@ if sdk.health().is_ok() {
 }
 ```
 
+{{% feature publishVersion="0.12.0" %}}
+
+To mark the [game session as ready]({{< relref "_index.md#ready" >}}) call `sdk.ready()`.
+
+```rust
+if sdk.ready()?
```

Comment:
I don't know much about rust, but this looks incomplete.

Reply:
Ah good catch. Will fix.

Reply:
Done!